pages/reference/public-api/export-branch-stats.mdx

```diff
@@ -0,0 +1,59 @@
+{/* cSpell:disable */}
+
+# Export Branch stats
+
+_Export branch stats using Avo API_
+
+The Export API gives you the possibility to export your Avo branches as CSV.
+
+## Endpoint
+
+```Url
+https://api.avo.app/workspaces/:workspaceId/branches/export/v1
+```
+
+`:workspaceId` is the ID of your workspace. You'll find it in the URL of your avo tab after `/schemas/`.
+
+Returns [a CSV string that is a representation of your branch stats]
+
+CSV Headers
+
+- `branchName`: Name of the branch
+- `branchId`: Id of the branch
+- `branchCreatorId`: userId of the branch creator
+- `branchCreatorEmail`: email of the branch creator
+- `creationDate`: UTC timestamp when the branch was created
+- `branchStatus`: current status on the branch [Merged | Closed | Approved | Draft | ReadyForReview]
+- `collaborators`: array of user emails that helped on the branch.
+- `lastPublishedDate`: UTC timestamp of last time when the branch was published to a downstream publishing integration
+- `markedReadyForReviewDate`: UTC timestamp when the branch was marked as ReadyForReview
```

Comment:
I assume this is the for the first time a branch is marked as ready for review, for cases where that is done multiple times on the same branch?

Reply:
yes